presentation/build.gradle.kts

```diff
@@ -40,32 +41,33 @@ android {
 }
 
 dependencies {
-  val composeVersion = "1.4.0-alpha04"
-  val coilVersion = "2.2.2"
+  val composeBom = platform(libs.androidx.compose.bom)
+  implementation(composeBom)
+  androidTestImplementation(composeBom)
 
   implementation(project(":shared"))
 
-  implementation("androidx.core:core-ktx:1.9.0")
-  //implementation("androidx.activity:activity-compose:1.6.1")
-  implementation("androidx.compose.ui:ui:$composeVersion")
-  debugImplementation("androidx.compose.ui:ui-tooling:$composeVersion")
-  implementation("androidx.compose.ui:ui-tooling-preview:$composeVersion")
-  implementation("androidx.compose.material:material:$composeVersion")
-  implementation("androidx.compose.material:material-icons-extended:$composeVersion")
-  // Material3 in Compose
-  implementation("androidx.compose.material3:material3:1.1.0-alpha04")
+  implementation(libs.androidx.core.ktx)
+  implementation(libs.androidx.compose.ui)
+  debugImplementation(libs.androidx.compose.ui.tooling)
+  implementation(libs.androidx.compose.ui.tooling.preview)
 
-  implementation("androidx.activity:activity-compose:1.6.1")
-  implementation("androidx.lifecycle:lifecycle-runtime-compose:2.6.0-alpha04")
-  // Image loading
-  implementation("io.coil-kt:coil:$coilVersion")
-  implementation("io.coil-kt:coil-compose:$coilVersion")
+  // TODO remove this????
+  //implementation("androidx.compose.material:material:$composeVersion")
```

Comment:
I think this was leftover remnants from my copy/pasted dependencies on my other project, and I *think* is fine to remove?

Reply:
agreed.  I think we should migrate to just using Material3 as much as possible.